libr/core/cmd_anal.c

```diff
@@ -5648,6 +5648,10 @@ static void _anal_calls(RCore *core, ut64 addr, ut64 addr_end, bool printCommand
 						isValidCall = false;
 					}
 				}
+				RBinReloc *rel = r_core_getreloc (core, addr, op.size);
+				if (rel && (rel->import || rel->symbol)) {
+					isValidCall = false;
+				}
 				if (isValidCall) {
 #if JAYRO_03
```

Comment:
i think this is what JAYRO_03 was  suposed to do, this was speedingi up the analysis on PE quite a biit afair. can you review that too because maybe we can remove this code now